Eshop/apps/notifications/tasks.py

```diff
@@ -0,0 +1,13 @@
+from celery import shared_task
+from django.core.mail import send_mail
+
+
+@shared_task
+def send_email_task():
```

Comment:
add email recipient+text+
```suggestion
def notify_user(system='email', **kwargs):
    if source == 'email':
         send_to_email()
```